css/style.css

```diff
@@ -0,0 +1,164 @@
+
+.body{
+  max-width: 1440px;
+  background: white;
+  margin: 0 60px 24px 42.72px;
+  font-family: "Montserrat";
+  color:#000000;
+}
+.header{
+  display: flex;
+  justify-content: space-between;
+  height: 109px;
+  align-items: center;
+}
+.header__img{
+  display: block;
+}
+.bloks-1{
+  flex-grow: 0.05;
+}
+.bloks-2{
+  flex-grow: 0.35;
+}
+.bloks-3{
+  flex-grow: 0.065;
+}
+.car__title{
+  text-align: center;
+  font-weight: 600;
+  font-size: 90px;
+  margin-top:0;
+  margin-bottom: 0;
+  font-slyle: normal;
+}
+.car__subtitle{
+  text-align: center;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 15px;
+  margin-top: 0;
+}
+.car__tsl{
+  display: block;
+  height: 450px;
+  margin: auto;
+}
+.car{
+  background: url('../IMG/tesla bgr.png') no-repeat;
+  height: 100%;
+}
+.header__menu{
+  display: flex;
+  justify-content: space-evenly;
+}
+.header__menu_link{
+  color: #000000;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  text-decoration: none;
+  line-height: 18px;
+}
+.header__menu_link-red{
+color: #e51937;
+text-decoration: underline;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)